fixtures/js-stubs/availableNotificationActions.js

```diff
@@ -1,119 +0,0 @@
-export function AvailableNotificationActions(params = []) {
```

Comment:
this Stub is being used in the test https://github.com/getsentry/sentry/blob/82e0ba1a4ffcdc7d9b1ee2a0bba40f0b661cfc33/static/app/components/notificationActions/notificationActionManager.spec.jsx#L14, therefore we can't remove it.

Reply:
Ups, adding back. I had *.tsx file filter, will convert this test to tsx as well